apps/admin/frontend/test/render_in_app_context.tsx

```diff
@@ -88,16 +88,7 @@ export function renderRootElement(
     // apiClient here, which will cause an error if the test tries to make an
     // API call.
     apiClient,
-    // TODO: Determine why tests fail when using useErrorBoundary = true
-    queryClient = new QueryClient({
-      defaultOptions: {
-        ...QUERY_CLIENT_DEFAULT_OPTIONS,
-        queries: {
-          ...(QUERY_CLIENT_DEFAULT_OPTIONS.queries ?? {}),
-          useErrorBoundary: false,
-        },
-      },
-    }),
+    queryClient = createQueryClient(),
```

Comment:
Thanks for getting to the bottom of this! Figured that was probably the case - that a real bug in a test was being masked